### PR TITLE
Adding join link for Slack.

### PR DESCRIPTION
**Pull request by gauravgahlot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
-# DockerPune.github.io
\ No newline at end of file
+# Docker Pune
+
+&nbsp;
+
+## Stay in Touch
+[Join us](https://bit.ly/2U2THw5) over **Slack** for:
+ - important blogs
```
Comment (sagar-jadhav) [inline README.md]: Formatting comment use capital first letter for e.g. **Important blogs**. As per my opinion benefits are:

1. Blogs
2. Free courses
3. Upcoming Meet ups/Events
4. Asking questions

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
-# DockerPune.github.io
\ No newline at end of file
+# Docker Pune
+
+&nbsp;
+
+## Stay in Touch
+[Join us](https://bit.ly/2U2THw5) over **Slack** for:
```
Comment (sagar-jadhav) [inline README.md]: Join us at [Docker Pune](https://bit.ly/2U2THw5) over **Slack** and get benefited for the following:

File: README.md [hunk 3]
```diff
@@ -1 +1,10 @@
-# DockerPune.github.io
\ No newline at end of file
+# Docker Pune
+
+&nbsp;
+
+## Stay in Touch
```
Comment (sagar-jadhav) [inline README.md]: It can be headlined as **Contact us**

Comment (gauravgahlot) [reply README.md]: I have kept it as is for now.

**Comment by gauravgahlot** [comment]
@sagar-jadhav Made the suggested changes.

File: README.md (hunk 3 above)
Comment (sagar-jadhav) [reply README.md]: Ok 👍

**Pull request merged**